libqfieldsync/offline_converter.py

```diff
@@ -102,7 +103,9 @@ def __init__(
         # elipsis workaround
         self.trUtf8 = self.tr
 
-        self.export_folder = Path(export_folder)
+        self.export_folder = Path(export_filename).parent
+        self._export_filename = Path(export_filename).stem
```

Comment:
```suggestion
        self._export_filename = Path(export_filename)
```

Can we explore if we can kill the `export_folder` attribute and use `self._export_filename.parent` instead?

Note that `_export_filename` has changed from the `stem` to the full filename. If we keep the previous state, we need to rename it to `_export_basename`.